### PR TITLE
Deadlines and Attendance

### PR DESCRIPTION
**Pull request by burgess01**
## Description

We completed the deadlines and attendance sections and want to add them to the main repositories. We added what you should and shouldn't do for these topics.

### Type of Change

- [ ] Feature
- [ ] Bug fix
- [x] Documentation

## Contributors

- @burgess01 @laurennevill @tuduun @MAbraham24

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

**Comment by mariakimheinert** [comment]
Hi @burgess01! Would you please add a description to your initial comment to give more information about the changes this PR contains?

**Comment by burgess01** [comment]
> Hi @burgess01! Would you please add a description to your initial comment to give more information about the changes this PR contains?

Whoops! I added the description in a commented out block. The description is now available!

**Review by solisa986 — DISMISSED** [review]
Everything looks good! Great work everyone on your first PR!

**Review by mariakimheinert — DISMISSED** [review]
Team 2, these changes look great! Thanks for all your hard work 🙂 :shipit:

**Review by PaigeCD — DISMISSED** [review]
Agreed and approved! ✨✨✨

File: conduct.md [hunk 1]
```diff
@@ -20,35 +20,39 @@ TODO: Write an introductory paragraph.
...
+- Students should be aware of deadlines and get assistance for assignments prior to end of deadlines.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should be mindful of assignment due dates and work on them accordingly.
+- Students should not request an extension unless they are facing a serious and extenuating circumstance.
+- Students who have a situation where they need an extension should contact the professor as soon as possible.
```
Comment (noorbuchi) [inline conduct.md]: The first and last point in this section do not follow the structure of unacceptable conduct where each bullet point begins with `Students should not`. Please make sure to rephrase this. Thank you!

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Great work overall! Thanks for all your contribution. Once the point I brought up is addressed, I will approve this PR.

File: conduct.md (hunk 1 above)
Comment (burgess01) [reply conduct.md]: I rephrased it and now it matched everything else. Thank you!

**Review by noorbuchi — APPROVED** [review]
Looks great! Thanks for addressing the requested changes. :shipit:

**Review by ullrichd21 — APPROVED** [review]

**Review by mariakimheinert — APPROVED** [review]

**Review by nathandloria — APPROVED** [review]

**Review by PaigeCD — APPROVED** [review]

**Review by anthonygc — APPROVED** [review]
Looks good! Everything seems very clear and concise.

**Review by watson02 — APPROVED** [review]
All good

**Pull request merged**